src/rgw/rgw_lc.cc

```diff
@@ -1414,10 +1414,6 @@ int LCOpRule::process(rgw_bucket_dir_entry& o,
 
     int r = (*selected)->process(ctx);
     if (r < 0) {
-      ldpp_dout(dpp, 0) << "ERROR: remove_expired_obj " 
```

Comment:
ok, so I guess "remove_expired_obj" is incorrect;  but why do we not want to alert on the error?  can you show how you know it's redundant?